extension/jaegerremotesampling/jaegerremotesamplingdeprecated/internal/sampling/strategystore/factory.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2018 The Jaeger Authors.
```

Comment:
I know it's only temporary but should the copied files should include the otel license at the top?

Reply:
Would they be added on top? Meaning that there would be otel + jaeger? Or how does it work?

Reply:
You can't remove the copyright from the original, but you can add ours to it, in case we do changes here.

Reply:
There are no changes planned. I hope it simply gets removed in the upcoming releases.